### PR TITLE
Fix to README

### PR DESCRIPTION
**Pull request by wltrup**
Changed version number to 1.0 in the dependency management sections of the README file.
Updated podspec version entry to 0.1.2.

~Changed version number from 0.0.1 to 0.1.1 in the dependency management sections of the README file.~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ If you use [Carthage][] to manage your dependencies, simply add
 DrawerKit to your `Cartfile`:
 
 ```
-github "DrawerKit/DrawerKit" ~> 0.0.1
+github "DrawerKit/DrawerKit" ~> 0.1.1
```
Comment (dmcrodrigues) [inline README.md]: We shouldn't lock on a patch level, this should either lock on `0.1` or `1.0` with my preference for the later.

File: README.md [hunk 2]
```diff
@@ -229,7 +229,7 @@ your Carthage framework copying build phase.
 If you use [CocoaPods][] to manage your dependencies, simply add DrawerKit to your `Podfile`:
 
 ```
-pod 'DrawerKit', '~> 0.0.1'
+pod 'DrawerKit', '~> 0.1.1'
```
Comment (dmcrodrigues) [inline README.md]: Ditto.

File: README.md (hunk 1 above)
Comment (wltrup) [reply README.md]: Fixed in https://github.com/Babylonpartners/DrawerKit/pull/21/commits/76e2ce3b2c5cb3ad7de5372b9191c080ffd4b67d.

File: README.md (hunk 2 above)
Comment (wltrup) [reply README.md]: Fixed in https://github.com/Babylonpartners/DrawerKit/pull/21/commits/76e2ce3b2c5cb3ad7de5372b9191c080ffd4b67d.

**Pull request merged**